### ISSUE TITLE
Add xml:lang on <p>

### ISSUE THREAD

**Issue by juliaflanders** [Digital-Humanities-Quarterly/dhq-journal#14]
Needed for multilingual examples

**Comment by juliaflanders** [comment]
When making this update, we should also add @xml:lang on <q>, <quote>, <said>, and <term>

**Comment by amclark42** [comment]
Accurately marking the language of textual content is an accessibility issue ([WCAG 2.2 success criterion 3.1.2](https://www.w3.org/WAI/WCAG22/Understanding/language-of-parts.html)). Among other uses, screen readers rely on language tags to determine how words should be pronounced.

In a recent article, @brgrey and I found that `@xml:lang` is _also_ needed on elements such as `<ab>`, `<label>`, `<note>`, and `<hi>`, and likely others. In order to mark that an example is in German, Benjamin could not place the `@xml:lang` on the part of the example in that language. Instead, he had to put the `@xml:lang` on the `<dhq:example>`, which is then lying about the language of the labels, legend, and note (which are in English). To override the German language code, we'd need to put `@xml:lang="en"` on `<label>`, `<note>`, and `<dhq:caption>`.

Below is a simplified example of what we ran into.

```xml
<dhq:example xml:id="example01" xml:lang="de">
<ab><label xml:lang="en">The preliminary categories:</label><!-- @xml:lang is invalid here --> 
  person, occupation, place, cause-of-death, age
  <lb/>
  <lb/> </ab>
<ab><label xml:lang="en">A sample annotated record:</label><!-- @xml:lang is invalid here --> 
  Der <hi>Johann Rendt</hi> / ein <hi>Zuckerbacher</hi> in
  <hi>Berdronischen Hauß</hi> in <hi>Offenloch</hi> ist an der <hi>Lungelsucht</hi>
  beschaut alt <hi>35. Jahr</hi>. (WD 08.08.1703:
  9)<note xml:lang="en"><!-- @xml:lang is invalid here --> 
    Here and in the following sections, text passages from the <title rend="italic">Wien[n]erisches
    Diarium</title> (WD) or from the <title rend="italic">Wiener Zeitung</title> (WZ) will be quoted with their date
    of publication and page number.</note></ab>
  <dhq:caption xml:lang="en"><!-- @xml:lang is invalid here -->
    <p>In the annotated record, “Johann Rendt” is coded as a “person”; “Zuckerbacher” is coded 
      as an “occupation”; “Berdronischen Hauß” and “Offenloch” are coded as “place”; 
      “Lungelsucht” is coded as “cause of death”; and “35. Jahr” is coded as “age”.</p>
  </dhq:caption>
</dhq:example>
```

It might be easier to make `@xml:lang` a global attribute, as the TEI does.

Unfortunately, all of this is purely theoretical because, as @sydb points out in #74, the DHQ stylesheets don't consistently put out `@xml:lang` in the HTML, even when the attribute is valid.
